cea/config.py

```diff
@@ -1104,7 +1106,7 @@ def parse_string_coordinate_list(string_tuples):
 def validate_coord_tuple(coord_tuple):
     """Validates a (lat, long) tuple, throws exception if not valid"""
 
-    lat, lon = coord_tuple
+    lon, lat = coord_tuple
```

Comment:
I'm not too sure why this is reversed. I thought that latitude was before longitude.